tests/h/services/group_test.py

```diff
@@ -5,14 +5,30 @@
 import mock
 import pytest
 
-from h.models import User, GroupScope
+from h.models import User, Group, GroupScope
 from h.models.group import ReadableBy
 from h.services.group import GroupService
 from h.services.group import groups_factory
 from h.services.user import UserService
 from tests.common.matchers import Matcher
 
 
+class TestGroupServiceFetch(object):
+
+    def test_it_returns_group_model(self, svc, factories):
+        group = factories.Group()
+
+        fetched_group = svc.fetch(group.pubid)
+
+        assert fetched_group == group
+        assert isinstance(fetched_group, Group)
```

Comment:
Is it possible for this second assert to fail after the first one has passed?

Reply:
No, but I was aiming for explicit communication to other devs more than actual test failure, if that makes sense